tests/test_msg_cosmos_signtx.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import common
+
+from base64 import b64encode
+from binascii import hexlify
+
+import keepkeylib.messages_pb2 as proto
+import keepkeylib.messages_stellar_pb2 as stellar_proto
+from keepkeylib.tools import parse_path
+
+DEFAULT_BIP32_PATH = "m/44h/118h/0h/0/0"
+
+class TestMsgCosmosGetAddress(common.KeepKeyTest):
+    def test_cosmos_sign_tx(self):
+        self.setup_mnemonic_nopin_nopassphrase()
+        signature = self.client.cosmos_sign_tx(parse_path(DEFAULT_BIP32_PATH), 19637, "cosmoshub-2", 5000, 200000, "cosmos15cenya0tr7nm3tz2wn3h3zwkht2rxrq7q7h3dj", "cosmos18vhdczjut44gpsy804crfhnd5nq003nz0nf20v", 100000, "", 3)
```

Comment:
Do these correspond to txs that happened on-chain? If so, please add a comment with the txid. If not, let's add some "real" on-chain tests in addition to these.